instances/widgets.treasury-factory.near/widget/pages/lockup/Table.jsx

```diff
@@ -0,0 +1,454 @@
+};
+const { decodeBase64 } = VM.require(
+  "${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/lib.common"
+);
+const { TableSkeleton } = VM.require(
+  "${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/lib.skeleton"
+);
+const { treasuryDaoID, lockupContract, allowLockupCancellation } = VM.require(
+  `${instance}/widget/config.data`
+);
+
+if (!instance || !TableSkeleton || !decodeBase64 || !treasuryDaoID)
+  return <></>;
+
+const columnsVisibility = JSON.parse(
+  Storage.get(
+    "COLUMNS_VISIBILITY",
+    `${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/pages.lockup.SettingsDropdown`
+  ) ?? "[]"
+);
+
+const [showToastStatus, setToastStatus] = useState(false);
+const [voteProposalId, setVoteProposalId] = useState(null);
+const [lockupNearBalances, setLockupNearBalances] = useState(null);
+const hasVotingPermission = (
+  transferApproversGroup?.approverAccounts ?? []
+).includes(accountId);
+const hasDeletePermission = (deleteGroup?.approverAccounts ?? []).includes(
+  accountId
+);
+
+const Container = styled.div`
+  font-size: 14px;
+  min-height: 60vh;
+  display: flex;
+
+  td {
+    padding: 0.5rem;
+    color: inherit;
+    vertical-align: middle;
+    background: inherit;
+  }
+
+  table {
+    overflow-x: auto;
+
+    thead td {
+      text-wrap: nowrap;
+    }
+  }
+
+  .text-warning {
+    color: var(--other-warning) !important;
+  }
+
+  .markdown-href p {
+    margin-bottom: 0px !important;
+  }
+
+  .markdown-href a {
+    color: inherit !important;
+  }
+`;
+
+function checkProposalStatus(proposalId) {
+  Near.asyncView(treasuryDaoID, "get_proposal", {
+    id: proposalId,
+  })
+    .then((result) => {
+      setToastStatus(result.status);
+      setVoteProposalId(proposalId);
+      refreshTableData();
+    })
+    .catch(() => {
+      // deleted request (thus proposal won't exist)
+      setToastStatus("Removed");
+      setVoteProposalId(proposalId);
+      refreshTableData();
+    });
+}
+
+useEffect(() => {
+  if (props.transactionHashes) {
+    asyncFetch("${REPL_RPC_URL}", {
+      method: "POST",
+      headers: {
+        "content-type": "application/json",
+      },
+      body: JSON.stringify({
+        jsonrpc: "2.0",
+        id: "dontcare",
+        method: "tx",
+        params: [props.transactionHashes, context.accountId],
+      }),
+    }).then((transaction) => {
+      if (transaction !== null) {
+        const transaction_method_name =
+          transaction?.body?.result?.transaction?.actions[0].FunctionCall
+            .method_name;
+
+        if (transaction_method_name === "act_proposal") {
+          const args =
+            transaction?.body?.result?.transaction?.actions[0].FunctionCall
+              .args;
+          const decodedArgs = JSON.parse(atob(args ?? "") ?? "{}");
+          if (decodedArgs.id) {
+            const proposalId = decodedArgs.id;
+            checkProposalStatus(proposalId);
+          }
+        }
+      }
+    });
+  }
+}, [props.transactionHashes]);
+
+useEffect(() => {
+  if (lockupContract)
```

Comment:
do we need lockup in table? 
user can't create a lockup creation request from lockup account.

Reply:
I still see lockup, we don't need it